cli/src/test/resources/fixture/cquery-test-android-code-change-android-impacted-targets.txt

```diff
@@ -1,3 +1,11 @@
+@@//src/main/java/com/integration:GuavaUserAndroid.java
```

Comment:
It's surprising this source file appears here, but it is not filtered out by the starlark function. From my observation, it does have providers:

> {"VisibilityProvider": <input file target //src/main/java/com/integration:GuavaUserAndroid.java>, "LicensesProvider": <input file target //src/main/java/com/integration:GuavaUserAndroid.java>, "FileProvider": <unknown object com.google.devtools.build.lib.analysis.FileProvider>, "FilesToRunProvider": <unknown object com.google.devtools.build.lib.analysis.FilesToRunProvider$SingleExecutableFilesToRunProvider>}

Reply:
I deleted that comment in a later commit